test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -1878,20 +1887,29 @@ protected static boolean isXPackTemplate(String name) {
             case "saml-service-provider":
             case "logs":
             case "logs-settings":
+            case "logs@settings":
             case "logs-mappings":
+            case "logs@mappings":
             case "metrics":
             case "metrics-settings":
+            case "metrics@settings":
             case "metrics-tsdb-settings":
+            case "metrics@tsdb-settings":
             case "metrics-mappings":
+            case "metrics@mappings":
             case "synthetics":
             case "synthetics-settings":
+            case "synthetics@settings":
             case "synthetics-mappings":
+            case "synthetics@mappings":
             case ".snapshot-blob-cache":
             case "ilm-history":
             case "logstash-index-template":
             case "security-index-template":
             case "data-streams-mappings":
+            case "data-streams@mappings":
             case "ecs@dynamic_templates":
+            case "ecs@mappings":
```

Comment:
Actually, we now have a convention, so simple and more future proof something like:
```
        if (name.contains("@")) {
            return true;
        }
```